kong/conf_loader/init.lua

```diff
@@ -1509,8 +1528,11 @@ local function load(path, custom_conf, opts)
   end
 
   if conf.lua_ssl_trusted_certificate then
-    conf.lua_ssl_trusted_certificate =
-      pl_path.abspath(conf.lua_ssl_trusted_certificate)
+    local abs_paths = {}
+    for i, path in ipairs(conf.lua_ssl_trusted_certificate) do
+      abs_paths[i] = pl_path.abspath(path)
+    end
+    conf.lua_ssl_trusted_certificate = abs_paths
```

Comment:
```suggestion
    conf.lua_ssl_trusted_certificate = pl_tablex.map(pl_path.abspath, conf.lua_ssl_trusted_certificate)
```

should be equivalent, handwritten not tested though.